client/document/browsehappy.jsx

```diff
@@ -1,33 +1,27 @@
 /**
```

Comment:
FWIW, I'm currently getting a 500 ("We're sorry, but an unexpected error has occurred") if I visit `http://calypso.localhost:3000/browsehappy` in incognito mode. (That's not a regression though, the same is true for https://wpcalypso.wordpress.com/browsehappy)